Author: Pino Toscano <pino@debian.org>
Description: Remove the "+git" suffix from the version string.
 It was forgotten by upstream when tagging the version.
Last-Updated: 2019-07-10
Forwarded: not-needed

--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -2,7 +2,7 @@
 cmake_minimum_required(VERSION 2.8.12 FATAL_ERROR)
 
 project(Tellico)
-set(TELLICO_VERSION "3.2.1+git")
+set(TELLICO_VERSION "3.2.1")
 
 set(QT_MIN_VERSION "5.4.0")
 
